### PR TITLE
Dev branch

### PR DESCRIPTION
**Pull request by akbarsazish**
Capstone project
#images, style.css and script files are inside of the assets folder.
#It has two pages index.html and about.html

### CODE REVIEW

**Comment by alexander16108** [comment]
Hi @akbarsazish , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by akbarsazish** [comment]
Dear @alexander16108 , I apologize for the linter errors in the code. I understand that it could have been avoided by using template literals instead of string concatenation. I will ensure to make the necessary adjustments and adhere to coding standards in the future.

File: index.html [hunk 1]
```diff
@@ -6,10 +6,142 @@
...
     <link rel="stylesheet" href="assets/style.css">
-    <script src="assets/script.js"></script>
 </head>
+
 <body>
-    
+    <header class="header">
+        <nav class="navbar top-nave">
+            <div class="log">
+                <a href="index.html" class="logo-link"> S.Academy </a>
```
Comment (Kossi-stack) [inline index.html:35]: - [ ] Kindly add the header where there is Facebook and Twitter.
- Your design
Screenshot 2023-05-31 at 09 16 10
- Behance Design
Screenshot 2023-05-31 at 09 22 13

File: index.html [hunk 2]
```diff
@@ -6,10 +6,142 @@
...
+        <div class="main-program-link">
+            <button class="join"> JOIN SAZISH ACADEMY  </button>
+            <a class="see-all-program" href="#">  <span class="see-all"> SEE ALL PROGRAM </span>  </a>
+        </div>
+    </section>
+
+    <!-- speaker section start -->
+    <section class="speakers">
+        <h1 class="speaker-headline">  Feature Speakers  </h1>
+           <hr>
```
Comment (Kossi-stack) [inline index.html:134]: - [ ] Kindly make sure the feature speakers pictures have the same size.
- Your design
Screenshot 2023-05-31 at 09 34 52

- Behance Design
Screenshot 2023-05-31 at 05 26 43

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @akbarsazish,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [ ] Kindly add a descriptive pull request message, This helps other programmers who want to take a look at your code to understand what changes you made to a particular branch, you can find an example [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (akbarsazish) [reply index.html:35]: Dear @Kossi-stack, I would like to express my gratitude for reviewing the code and providing your recommendations. I appreciate your valuable insights, and I will make sure to take them into consideration. Thank you for your time and effort in reviewing the code.

File: index.html [hunk 3]
```diff
@@ -9,6 +9,26 @@
...
+                <a href="" class="language"> Lanuage </a>
+            </li>
+            <li class="contact-item">
+                <a href="" class="mypag"> My Page </a>
+            </li>
+            <li class="contact-item">
+                <a href="" class="logout"> logout </a>
+            </li>
+          
+        </ul>
```
Comment (AbayomiOlaoye) [inline index.html:30]: - [ ] I must commend your design skills, your page looks really nice, and thanks to the previous reviewer, your looks better now... :clap:  The only issue is that your additional nav bar is causing a design conflict that is currently not too good for user experience. How about hiding it completely when the screen width is `< 768px`? Or better still, you can add it to the main menu in your mobile view :wink: 

image

- [ ] [OPTIONAL] Your hamburger menu doesn't have a close option to collapse the menu should the user decide not to click on any button. I think this is a great feature that can further increase user experience :+1: 
image

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @akbarsazish :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so_close


 ## Highlights
- Good use of HTML semantic tags ✅ 
- Responsive page ✅ 
- I love your design ✅ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (akbarsazish) [reply index.html:30]: Hi @AbayomiOlaoye, In my computer I hide the nave bar (top nav) and also mobile menu has the close button, everythings is working pefeclty. Please press ctrl + f5 on computer or somthing else. 
As I don't have the cases that mentioned I don't know what to do, as you know it is the capstone project I have to outline the presentation and record the vidoe so, my time is limited. 

please let me know what should I do.

top nav is hide
close btn

Comment (akbarsazish) [reply index.html:30]: Should I submit the pull request again or ???

Comment (akbarsazish) [reply index.html:30]: .top-header,
.hide-btn {
  display: none;
}

the above code hide that menu while it is  > 768px

Comment (AbayomiOlaoye) [reply index.html:30]: @akbarsazish, Now, that's a real issue. It does meant that it has worked before. Kindly clear your browser's cache memory or try it on another browser. If everything checks well, you can submit again

Comment (akbarsazish) [reply index.html:30]: @AbayomiOlaoye, I have cleared the browser cache and checked with two different browser it works.

**Comment by AbayomiOlaoye** [comment]
Then you can make a new request 😉. Good job

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @akbarsazish 🙋‍♀️

Great work implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Your README looks good, well done!
-  ✅ Commits are descriptive, good work.
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**